OracleTuxedo/README.md

```diff
@@ -5,6 +5,7 @@ This repository contains the container image configuration, build and sample fil
 The following container image is available:
 
 * [Tuxedo Core](./core) Tuxedo core including server binaries and client libraries
+* [Tuxedo SALT](./salt) Tuxedo SALT bankapp sample application
```

Comment:
```suggestion
* [Oracle Service Architecture Leveraging Tuxedo (SALT)](./salt) sample ATMI banking application
```

Reply:
Changed to 
 * [Oracle Service Architecture Leveraging Tuxedo (SALT)](./salt) Sample banking application

Reply:
You need to push those changes to GitHub as this fix and all the others you've mentioned have not yet been updated. Note that when you do push changes to address review comments, you don't have to reply to those comments: GitHub will automatically flag the comment as resolved if the line mentioned has been modified.